packages/pcd/gpc-pcd/src/GPCPCDPackage.ts

```diff
@@ -274,39 +272,49 @@ export function getProveDisplayOptions(): ProveDisplayOptions<GPCPCDArgs> {
       pods: {
         argumentType: ArgumentTypeName.Record,
         description: "Generate a proof for the selected POD object",
-        validate:
-          (podName) =>
-          (podPCD, params): boolean => {
-            if (podPCD.type !== PODPCDTypeName) {
+        validate: (podName, podPCD, params): boolean => {
+          if (podPCD.type !== PODPCDTypeName) {
+            return false;
+          }
+
+          if (params?.proofConfig !== undefined) {
+            const proofConfig = ((): GPCProofConfig | undefined => {
```

Comment:
I understand the use of the lambda here, but what's its value?
I think if the try/catch was directly in this block the `return false` could be part of the catch.
The only thing which would have to change is to make the `proofConfig` declaration a let rather than a const so it can be set inside the try.

Reply:
Sorry, force of habit. I tend to avoid the pattern `let x; x = ...;`. I'll make the change!